src/fsharp/nameres.fs

```diff
@@ -2452,8 +2452,17 @@ let ResolveRecordOrClassFieldsOfType (ncenv: NameResolver) m ad typ statics =
     |> List.filter (fun rfref -> rfref.IsStatic = statics  &&  IsFieldInfoAccessible ad rfref)
     |> List.map Item.RecdField
 
+[<RequireQualifiedAccess>]
+type ResolveCompletionTargets =
+    | All of (MethInfo -> TType -> bool)
+    | SettablePropertiesAndFields
+    member this.ResolveAll = 
```

Comment:
is `this.ResolveAll` really usefull?
lot of use are `if completionTargets.ResolveAll then ... else []` as function return (last line), easy to use `match`

when used like
```fsharp
match pinfo with
| Some(Item.Event(einfo)) -> completionTargets.ResolveAll && IsStandardEventInfo
```

can be used 
```fsharp
match pinfo, completionTargets with
| Some(Item.Event(einfo)), All _ -> IsStandardEventInfo ..
| _, SettablePropertiesAndFields -> false
```